ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -97,12 +106,14 @@ export default function CustomSpendingCap({
     getInputTextLogic(value).description,
   );
 
-  const handleChange = (valueInput) => {
+  const handleChange = async (valueInput) => {
```

Comment:
In `token-allowance.js` we have:
```
          <CustomSpendingCap
            txParams={txData?.txParams}
```
so `txParams` can be undefined

should we exit this `handleChange` function early if that is the case?